src/components/TextInput.tsx

```diff
@@ -24,7 +25,10 @@ const TextInput = ({
       type="text"
       {...props}
       placeholder={placeholder ?? "placeholder..."}
-      className={`bg-white px-3 py-2 min-w-[250px] w-fit font-normal rounded-lg border border-accent text-text ${variantClasses} ${className}`}
+      className={twMerge(
```

Comment:
This new function will merge tailwind classes. You can read more about why this is usually required in a tailwind project down below:

https://github.com/dcastil/tailwind-merge